WcaOnRails/app/views/server_status/index.html.erb

```diff
@@ -58,11 +58,8 @@
     </div>
   </div>
 
-  <div>
-    PID <%= Process.pid %>
-  </div>
+  <%= alert :info, "PID #{Process.pid}" %>
+  <%= alert :info, "Uptime #{duration_to_s(Time.now - WcaOnRails::BOOTED_AT)}" %>
```

Comment:
That uses a lot of space. Then again it looks weird without it.

I'd feel better if they shared a Big Blue Blob: `PID 59242 · Uptime: 28h 43m 14.03s`.